### PR TITLE
Update README.md on Whisper Python Demo

### PR DESCRIPTION
**Pull request by raymondlo84**
updated the README to reflect changes to the repos.

### CODE REVIEW

File: samples/python/whisper_speech_recognition/README.md [hunk 1]
```diff
@@ -15,9 +15,18 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
 
 ## Prepare audio file
 
-Prepare audio file in wav format with sampling rate 16k Hz.
+You can prepare an audio file in WAV format with a sampling rate of 16k Hz using the `recorder.py` script. The script will record 5 seconds of audio from the microphone.
```
Comment (Wovchena) [inline samples/python/whisper_speech_recognition/README.md:18]: `recorder.py` isn't added

Comment (Wovchena) [reply samples/python/whisper_speech_recognition/README.md:18]: I see now. The script is in https://github.com/openvinotoolkit/openvino.genai/pull/1010

Comment (Wovchena) [inline samples/python/whisper_speech_recognition/README.md:18]:
```suggestion
You can prepare an audio file in WAV format with a sampling rate of 16k Hz using the [`recorder.py`](recorder.py) script. The script will record 5 seconds of audio from the microphone. 
```

Comment (Wovchena) [inline samples/python/whisper_speech_recognition/README.md:18]: Please, add the same to C++ README. But the link to recorder.py should point to python folder.

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/python/whisper_speech_recognition/README.md (hunk 1 above)
Comment (raymondlo84) [reply samples/python/whisper_speech_recognition/README.md:18]: Sure

Comment (raymondlo84) [reply samples/python/whisper_speech_recognition/README.md:18]: Let's merge this one first? I will open another one for C++ side.

**Review by raymondlo84 — COMMENTED** [review]
Thanks!

**Review by Wovchena — APPROVED** [review]

File: samples/python/whisper_speech_recognition/README.md (hunk 1 above)
Comment (Wovchena) [reply samples/python/whisper_speech_recognition/README.md]: You could have already done that. But ok, let it be in another PR.

**Review by as-suvorov — APPROVED** [review]

**Comment by Wovchena** [comment]
build_jenkins

**Pull request merged**